### PR TITLE
 Implement List for Service

### PR DESCRIPTION
**Pull request by krhubert**
Dep on: https://github.com/mesg-foundation/engine/pull/1072

### CODE REVIEW

**Comment by antho1404** [comment]
why using a stream here. We don't have any pub/sub system for services yet so there is no reason to have a stream here. Let's simplify and return a list of services directly and let grpc/protobuf handle that

**Comment by NicolasMahe** [comment]
@krhubert I agree with Anthony. A stream seems overkill for now.. Let's develop the simplest api we can.

File: server/grpc/api/service.go [hunk 1]
```diff
@@ -63,9 +63,24 @@ func (s *ServiceServer) Get(ctx context.Context, req *protobuf_api.GetServiceReq
...
+// List returns all services.
+func (s *ServiceServer) List(ctx context.Context, req *protobuf_api.ListServiceRequest) (*protobuf_api.ListServiceResponse, error) {
+	services, err := s.sdk.Service.List()
+	if err != nil {
+		return nil, err
+	}
+
+	resp := &protobuf_api.ListServiceResponse{}
+	for _, service := range services {
+		resp.Services = append(resp.Services, ToProtoService(service))
```
Comment (NicolasMahe) [inline server/grpc/api/service.go]: I think there is a `ToProtoService` function. Let's use it or remove it.

Comment (antho1404) [reply server/grpc/api/service.go]: `mapping.go` > `ToProtoServices`

**Pull request merged**